### PR TITLE
Fix typo in assertEmptyThrowsMultipleFailureExceptionForManyThrowables()

### PR DESCRIPTION
**Pull request by ar-vargas**
at MultipleFailureExceptionTest.

I had problems with the other branch and I had to delete it.
Sorry for opening a new pull request.

### CODE REVIEW

File: src/main/java/org/junit/runner/JUnitCommandLineParseResult.java [hunk 1]
```diff
@@ -15,11 +15,6 @@
     private final List<Throwable> parserErrors = new ArrayList<Throwable>();
 
     /**
-     * Do not use. Testing purposes only.
-     */
-    JUnitCommandLineParseResult() {}
```
Comment (kcooney) [inline src/main/java/org/junit/runner/JUnitCommandLineParseResult.java]: This is not redundant. It is a package-scope constror, preventing code outside of this package from constructing an instance via the constructor.

**Comment by stefanbirkner** [comment]
I would like to merge ff7311b only. @kcooney  Do you agree?

**Comment by marcphilipp** [comment]
+1

**Comment by kcooney** [comment]
I agree with @stefanbirkner

**Comment by marcphilipp** [comment]
I cherry-picked ff7311b and pushed it to master.

@ar-vargas Thanks!

**Pull request closed without merging**